reminder_bot.py

```diff
@@ -0,0 +1,81 @@
+import telebot
+import sqlite3
+import datetime
+import csv
+
+# from first_sql_with_class import TelegrammBot
+# from get_function.reminder_func import get_reminder_days
+
+bot = telebot.TeleBot('5643692181:AAEvd665ZvwANdX1vcuf-T4eZVvk7CDBw3Y')
+
+
+@bot.message_handler(commands=['start'])
+def send_welcome(message):
+    response_message = f'Hello, {message.from_user.first_name} {message.from_user.last_name}\n'
+    bot.send_message(message.chat.id, response_message)
+
+
+@bot.message_handler(commands=['list'])
+def add_table_db(message):
+    connection = sqlite3.connect('date_for_tg.db')
+    cursor = connection.cursor()
+    cursor.execute("CREATE TABLE IF NOT EXISTS 'reminder' (ID INTEGER PRIMARY KEY AUTOINCREMENT, user_id INTEGER, user_date DATE)")
+    response_message = bot.reply_to(message, f'Список для добавления важных дат создан, введите дату')
+    bot.register_next_step_handler(response_message, add_msg_user_to_db)
+    connection.commit()
+
+
+def add_msg_user_to_db(message):
+    response_message = bot.reply_to(message, "your date is add\n")
+    bot.register_next_step_handler(response_message, get_msg_user_from_db)
+    connection = sqlite3.connect('date_for_tg.db')
+    cursor = connection.cursor()
+    # text = message.text
+    user_data = message.text
+    user_id = message.chat.id
+    cursor.execute("INSERT INTO reminder (user_id, user_date) VALUES (?, ?)", (user_id, user_data))
```

Comment:
as I know you should insert three columns: user_id - date time - date description